### PR TITLE
[ZEPPELIN-3510] Angular ui-grid fix

### PR DESCRIPTION
**Pull request by Savalek**
### What is this PR for?
This PR change display of spaces in ui-grid (Now the number of displayed spaces is correct).
And now then you edit the cell its displayed corectly.

### What type of PR is it?
Bug Fix 

### What is the Jira issue?
[ZEPPELIN-3510](https://issues.apache.org/jira/browse/ZEPPELIN-3510)

### <p align="center">Spaces</p>
Before :(
space_before

After :)
space_after

### <p align="center">Edit cell</p>
Before :(
edit_cell_before

After :)
edit_cell_after

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: zeppelin-web/src/app/notebook/paragraph/result/display-table.css [hunk 1]
```diff
@@ -34,10 +34,20 @@
...
+  white-space: pre-wrap !important;
   border-bottom: 1px solid #e8e8e8;
 }
 
+input[class *=coltuiGrid] {
+  white-space: pre-wrap !important;
+}
+
+.ui-grid-zeppelin-special-textarea {
+  width: -webkit-fill-available;
```
Comment (felixcheung) [inline zeppelin-web/src/app/notebook/paragraph/result/display-table.css]: does this only work with a particular family of browsers?

**Comment by Savalek** [comment]
@felixcheung, thanks. I fixed it.

**Comment by jongyoul** [comment]
LGTM

**Comment by jongyoul** [comment]
Will merge it

**Pull request closed without merging**